pilot/pkg/bootstrap/mesh.go

```diff
@@ -35,6 +35,15 @@ const (
 )
 
 // initMeshConfiguration creates the mesh in the pilotConfig from the input arguments.
+// Original/default behavior:
+// - use the mounted file, if it exists.
+// - use istio-REVISION if k8s is enabled
+// - fallback to default
+//
+// If the 'SHARED_MESH_CONFIG' env is set (experimental feature in 1.10):
+// - if a file exist, load it - will be merged
```

Comment:
merge order is file > istio-REV > SHARED_MESH_CONFIG?

Reply:
yes. 

File ( currently not used in OSS templates ) will have 'external istiod' overrides. 
And istio-rev should override the common ones.

Reply:
Will fix the comments.

Reply:
isn't the file part pretty important for external istiod?

Reply:
No, we're not using it right now ( since the code will not allow user to set it). It was an idea early on.

We may have a few 'forced vendor overrides' in our private branch.

Eventually we could merge it as well - but I wanted to keep it as simple as possible for the initial draft, get feedback - and 
if it turns to be useful we can expand. Else - revert. There are few other ideas discussed around mesh config future.

Reply:
can you update the comment?